src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php

```diff
@@ -11,13 +11,18 @@
 
 namespace Symfony\Component\Messenger\Handler;
 
-use Symfony\Component\Messenger\Exception\LogicException;
-
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
 trait BatchHandlerTrait
 {
+    /**
+     * Completes the jobs in the list.
+     *
+     * @list<array{0: object, 1: Acknowledger}> $jobs A list of pairs of messages and their corresponding acknowledgers
+     */
+    abstract private function process(array $jobs): void;
```

Comment:
methods go after properties, aren't?

Reply:
Not with fabbot.io

Reply:
They should yes

Reply:
And we never reorganize the code as it makes diff more difficult to read and merge more complex

Reply:
I can create a new pull request with the correct order if you want.

Reply:
yes, please :)

Reply:
The new PR : https://github.com/symfony/symfony/pull/44287